components/match2/wikis/apexlegends/match_summary.lua

```diff
@@ -463,7 +463,10 @@ function CustomMatchSummary._createOverallPage(match)
 						:addClass('panel-content__game-schedule__title')
 						:wikitext('Game ', idx, ':')
 						:done()
-				:node(CustomMatchSummary._gameCountdown(game))
+				:tag('div')
+					:addClass('panel-content__game-schedule__container')
```

Comment:
Several of these classes seems undefined, do we really need them? They take up bytes which is something we are trying to minimize

Reply:
I removed one html element that I found wasn't used. If you happen to know of more please mention this in https://gitlab.com/teamliquid-dev/liquipedia/web/issue-bucket/-/issues/44